master/buildbot/changes/bitbucket.py

```diff
@@ -116,6 +116,7 @@ def _processChanges(self, page):
                     # access additional information
                     author = pr['author']['display_name']
                     prlink = pr['links']['html']['href']
+                    committer = None
```

Comment:
We probably don't need this variable here, do we? Is it used later?

Reply:
Removed